vue/components/ui/molecules/input-currency/input-currency.vue

```diff
@@ -109,7 +109,7 @@ export const InputCurrency = {
         },
         _places(currency, fallback = 2) {
             const currencyInfo = currencies[currency] || {};
-            return currencyInfo.places || fallback;
+            return "places" in currencyInfo ? currencyInfo.places : fallback;

```

Comment:
Your approach is sound, but please change to be similar with the way we usually do this kind of stuff: `currencyInfo.places === undefined ? fallback : currencyInfo.places`